src/sidebar/components/ShareDialog/test/ExportAnnotations-test.js

```diff
@@ -374,7 +374,10 @@ describe('ExportAnnotations', () => {
     });
   });
 
-  it(
+  // FIXME This test fails after the change we did in SelectNext, to make the
+  //  toggle button have [role="combobox"].
+  //  Skipping for now, but we should get back to this ASAP.
+  it.skip(
```

Comment:
More context here https://hypothes-is.slack.com/archives/C1M8NH76X/p1699347560907199

I'll be addressing this first thing after merging this PR.

Reply:
I'm finally fixing this first https://github.com/hypothesis/client/pull/5940

I will rebase then here.